components/BuildPage/WaysToGrow.jsx

```diff
@@ -0,0 +1,130 @@
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { Card, CardTitle } from 'components/ui/card';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useState } from 'react';
+import Content from './Content';
+
+const ways = [
+  {
+    title: 'Earn Dev Rewards by building on Olas Protocol',
+    imageSrc: '/images/build-page/earn-dev-rewards.jpg',
+    description: (
+      <>
+        <p className="mb-4">
+          Contribute valuable code units — like agents or components — to the
+          Olas protocol and have a chance at receiving Developer Rewards.
+        </p>
+        <p className="mb-4">
+          Dev Rewards is a part of the protocol that facilitates the
+          distribution of capital to developers who contribute to various
+          services in the ecosystem.
+        </p>
+        <p>
+          This system is designed to reward both the contribution of code
+          components and entire agents.
+        </p>
+      </>
+    ),
+    linkAction: (setDevRewards) => (
+      <a
+        className="text-purple-600 cursor-pointer"
+        onClick={() => setDevRewards(true)}
+      >
+        Learn more about Dev Rewards ↗
+      </a>
+    ),
+  },
+  {
+    title: 'Explore opportunities with external projects',
+    imageSrc: '/images/build-page/explore-opportunities.jpg',
+    description: (
+      <>
+        Collaborate with external projects seeking skilled developers and
+        discover opportunities that align with your expertise, with compensation
+        handled directly by the third-party projects. <br />
+        <h3 className="font-semibold mt-4">Browse opportunities</h3>
+        Discover a list of projects that match your skills. <br />
+        <h3 className="font-semibold mt-4">Get in touch</h3>
+        Find projects that align with your expertise and interests.
+      </>
+    ),
+    link: (
+      <Link
+        href={'https://build.olas.network/opportunities'}
+        className="text-purple-600"
+      >
+        Get matched with an opportunity ↗
+      </Link>
+    ),
+  },
+];
+
+export const WaysToGrow = () => {
+  const [devRewards, setDevRewards] = useState(false);
```

Comment:
normally we name such states something like `[isDevRewardsOpen, setDevRewardsOpen]`, for now it sounds like we manage a list of dev rewards